skimage/measure/tests/test_regionprops.py

```diff
@@ -63,14 +63,14 @@ def test_bbox():
 def test_moments_central():
     mu = regionprops(SAMPLE)[0].moments_central
     # determined with OpenCV
-    assert_almost_equal(mu[0,2], 436.00000000000045)
+    assert_almost_equal(mu[0, 2], 436.00000000000045)
     # different from OpenCV results, bug in OpenCV
-    assert_almost_equal(mu[0,3], -737.333333333333)
-    assert_almost_equal(mu[1,1], -87.33333333333303)
-    assert_almost_equal(mu[1,2], -127.5555555555593)
-    assert_almost_equal(mu[2,0], 1259.7777777777774)
-    assert_almost_equal(mu[2,1], 2000.296296296291)
-    assert_almost_equal(mu[3,0], -760.0246913580195)
+    assert_almost_equal(mu[0, 3], -737.333333333333)
+    assert_almost_equal(mu[1, 1], -87.33333333333303)
+    assert_almost_equal(mu[1, 2], -127.5555555555593)
+    assert_almost_equal(mu[2, 0],  1259.7777777777774)
```

Comment:
Yes, indeed! I broke it in three places where arrays would have printed
rather badly otherwise.
On Dec 12, 2015 06:26, "Josh Warner" <notifications@github.com> wrote:

> In skimage/measure/tests/test_regionprops.py
> <https://github.com/scikit-image/scikit-image/pull/1815#discussion_r47432151>
> :
>
> >      # different from OpenCV results, bug in OpenCV
> > -    assert_almost_equal(mu[0,3], -737.333333333333)
> > -    assert_almost_equal(mu[1,1], -87.33333333333303)
> > -    assert_almost_equal(mu[1,2], -127.5555555555593)
> > -    assert_almost_equal(mu[2,0], 1259.7777777777774)
> > -    assert_almost_equal(mu[2,1], 2000.296296296291)
> > -    assert_almost_equal(mu[3,0], -760.0246913580195)
> > +    assert_almost_equal(mu[0, 3], -737.333333333333)
> > +    assert_almost_equal(mu[1, 1], -87.33333333333303)
> > +    assert_almost_equal(mu[1, 2], -127.5555555555593)
> > +    assert_almost_equal(mu[2, 0],  1259.7777777777774)
>
> I assume this pep8 violation is deliberate ;)
>
> (For the record I support it, especially in the test suite, but had to
> point it out)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-image/scikit-image/pull/1815/files#r47432151>.
>